packages/manager/cypress/support/util/random.ts

```diff
@@ -226,3 +226,23 @@ export const randomUuid = (): string => {
     randomString(12, randomStringOptions),
   ].join('-');
 };
+
+/**
+ * Returns a random secret of fixed length.
+ *
+ * @param length - Length of the secret strings.
+ *
+ * @returns Random secret.
+ */
+export const randomSecret = (length: number = 64): string => {
+  const hexNumber = '0123456789abcdef';
+
+  const characterSelection = [hexNumber].join('').split('');
```

Comment:
```suggestion
  const characterSelection = hexNumber.split('');
```
I think this could be simplified a bit! Is there anything major that distinguishes this from the `randomString()` util? If I'm understanding correctly we might be able to use `randomString(64)`